agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/configuration/Configuration.java

```diff
@@ -308,24 +379,24 @@ public String toString() {
     }
 
     public static class ProcessorIncludeExclude {
-        public ProcessorMatchType matchType;
+        public MatchType matchType;
         public List<String> spanNames = new ArrayList<>();
         public List<String> logNames = new ArrayList<>();
         public List<ProcessorAttribute> attributes = new ArrayList<>();
 
         public void validate(ProcessorType processorType, IncludeExclude includeExclude) throws FriendlyException {
             if (matchType == null) {
                 throw new FriendlyException(processorType.anX + " processor configuration has an " + includeExclude + " section that is missing a \"matchType\".",
-                        "Please provide \"matchType\" under the include/exclude section of the " + processorType + " processor configuration. " +
+                        "Please provide a \"matchType\" under the " + includeExclude + " section of the " + processorType + " processor configuration. " +
```

Comment:
includeExclude, typo?

Reply:
this is a variable `includeExclude` whose `toString()` will return one of "include" or "exclude"